application/controllers/User.php

```diff
@@ -200,7 +200,8 @@ function add() {
 				$data['user_callsign'] = $this->input->post('user_callsign');
 				$data['user_locator'] = $this->input->post('user_locator');
 				$data['user_timezone'] = $this->input->post('user_timezone');
-				$data['user_measurement_base'] = $this->input->post('user_measurement_base');
+				$data['user_measurement_base'] = $this->input->post('user_measurement_base') ?? 'Y';
```

Comment:
This needs correction. Measurement base can be "K", "M" or "N" (or none). But "Y" does not make sense here. Mistake?

![image](https://github.com/user-attachments/assets/dbc1d651-5d51-4b11-a03f-c209c274f17e)

Reply:
I guess we should rather have "K" or "M" here instead of "Y".